ledger/Cargo.toml

```diff
@@ -26,7 +26,7 @@ num_cpus = "1.0.0"
 rand = "0.6.5"
 rand_chacha = "0.1.1"
 rayon = "1.3.0"
-reed-solomon-erasure = { package = "solana-reed-solomon-erasure", version = "4.0.1-3", features = ["simd-accel"] }
```

Comment:
I don't know where this `"...-3"` thing in the version came from other than the guess we used this branch https://github.com/solana-labs/reed-solomon-erasure/tree/solana-4.0.1. Maybe we can just ignore that and go straight to `4.0.2` or is there any thing left?

Reply:
I think we just used the -X suffix when we published new versions based off the 4.0.1 upstream version.

Reply:
I see! thanks for explanation. Yet, the number of [`-3` and prior releases](https://crates.io/crates/solana-reed-solomon-erasure/versions) (= `3`) and the number of commits in the that branch (= `2`) doesn't match... Maybe retied republishes after failure?

I think this is a minor concern, though.